src/lengthEq.js

```diff
@@ -0,0 +1,25 @@
+import { equals } from 'ramda';
+
+import compareLength from './internal/compareLength';
+
+/**
+ * Returns `true` if the supplied list or string has a length equal to `valueLength`.
+ *
+ * @func lengthEq
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.8.0|v2.8.0}
+ * @category List
+ * @sig Number -> [*] -> Boolean
+ * @param {number} valueLength The length of the list or string
+ * @param {number} value The list or string
```

Comment:
`@param {Array|string} value ...`

This also apply to all the `length*` functions that you did before. Include changes to these function into this PR.